### PR TITLE
I add OIDs for pdu raritan and sensor controller from rittal CMC-III

### PR DESCRIPTION
**Pull request by Tarot67**
I tested with prometheus and grafana it's work. i decide to share. simply...

### CODE REVIEW

File: generator/generator.yml [hunk 1]
```diff
@@ -355,3 +355,40 @@ modules:
...
       - picoIPv4MIB
       - picoIPv6MIB
+# Raritan MIB
+#
+# Used by Legran/Raritan PDUs
+#
+# http://support.raritan.com/px2/version-2.4.1/mibs/pdu2-mib-020400-39592.txt
+  raritan_pdu:
+    walk:
+      - 1.3.6.1.4.1.13742.6.5.2.4.1.4.1.1 # input voltage/current etc...  mono/tri phase(s)
```
Comment (brian-brazil) [inline generator/generator.yml]: Please follow the existing pattern here

File: generator/generator.yml [hunk 2]
```diff
@@ -355,3 +355,40 @@ modules:
...
+      - 1.3.6.1.4.1.13742.6.3.3.6.1.7.1.1 # input voltage/current decimal for each precedents entrys
+      - 1.3.6.1.4.1.13742.6.5.2.3.1.4.1   # input statistic
+      - 1.3.6.1.4.1.13742.6.3.3.4.1.7.1   # input statistic decimal
+      - 1.3.6.1.4.1.13742.6.5.4.3.1.4     # outlet voltage/current etc...
+      - 1.3.6.1.4.1.13742.6.3.5.4.1.7.1   # outlet decimal
+
+    version: 2
```
Comment (brian-brazil) [inline generator/generator.yml]: These are defaults, no need to specify them.

File: generator/generator.yml (hunk 1 above)
Comment (Tarot67) [reply generator/generator.yml]: You prefers have the oid like : PDU2-MIB::measurementsOutletSensorValue.$reglette_num.1.1 ?

File: generator/generator.yml (hunk 2 above)
Comment (Tarot67) [reply generator/generator.yml]: you talk about version of snmp ?

**Comment by Tarot67** [comment]
for the make parse_errors i don't now why it's doesn't work. Because the file are here at good place!!!. And yes it exist!.

File: generator/generator.yml (hunk 1 above)
Comment (brian-brazil) [reply generator/generator.yml]: The name of the object would be usual.

File: generator/generator.yml (hunk 2 above)
Comment (brian-brazil) [reply generator/generator.yml]: Version and credentials.

File: generator/generator.yml (hunk 1 above)
Comment (Tarot67) [reply generator/generator.yml]: Ok no worry. I don't have time today. But tomorrow i change and test

File: generator/generator.yml (hunk 2 above)
Comment (Tarot67) [reply generator/generator.yml]: OK! I remove this options. No worry. Tomorrow

Comment (Tarot67) [reply generator/generator.yml]: Sorry for the time. I don't have access to my prometheus server and on my snmp pdu and devices. I can't change for the momment. I make change the 19 aout after my holiday.

**Comment by Tarot67** [comment]
Ok a made all MIBs variable and i remove ISO OIDs. I hope it's ok now.

Regards,

JeanSeb

**Comment by Tarot67** [comment]
There are a problem with several thing:

First the SNMPv2-MIB of Cisco don't respect the official MIB provide by IETF
i replace with Makefile of generator by good file.
Because the MIB file of RITTAL-CMC-III-MIB use official mib file and cisco rename some variables and this variables renamed doesn't exist in official MIB file ???

Two the last test i don't understand the error about snmp.yml ??? 

Thank! 

i need your help to resolve some conflicts

regard,

JeanSeb

**Comment by SuperQ** [comment]
Ping, it would be nice to update this. It might also be worth rebase + squash your commits.

**Comment by Tarot67** [comment]
Ok i am basic user with git. But with your explainations. I have some expert in my office. I try to resolve this. 
(I can add again ups (trademark legrand) and rittal chiller and so on... .When it work's well our prometheus/grafana i re pull request when it tested.

Thank you for your help

**Comment by SuperQ** [comment]
This seems abandoned.

**Pull request closed without merging**